Sources/PubNub/PubNub.swift

```diff
@@ -32,13 +32,11 @@ public struct PubNub {
   /// Instance identifier
   public let instanceID: UUID
   /// A copy of the configuration object used for this session
-  public let configuration: PubNubConfiguration
+  public private(set) var configuration: PubNubConfiguration
```

Comment:
I kind of liked the idea of immutable (that's my assumption) PubNubConfiguration and I'd love to keep it that way. I'm actually jealous because it's not immutable in java :(

Reply:
@kleewho users can’t change it. It can be changed only within this PubNub struct instance.